Neo4j.Driver/Neo4j.Driver/Internal/Routing/RoutingTableManager.cs

```diff
@@ -287,7 +287,7 @@ public IRoutingTable UpdateRoutingTable(ISet<Uri> triedUris = null, Func<IConnec
                     catch (Exception e)
                     {
                         _logger?.Info(
```

Comment:
This feels more like `Warn` than `Info` and can receive exception, not just the message